test/importers/test_import_openapi.py

```diff
@@ -0,0 +1,1289 @@
+      - in: header
+        name: metadata
+        schema:
+          type: boolean
+        required: false
+    get:
+      summary: Gets a user by ID
+      responses:
+        '200':
+          description: OK
+parameters:
+  id:
+    in: path
+    name: id
+    schema:
+      type: integer
+    required: true
+    description: The user ID.
+"""
+
+OPENAPI_WITH_PATHS_VAR_REFERRING_GLOBAL_PARAMS_OBJECT_EXPECTED_SYSL = r"""
+ "Sample API" [package=""]:
+    @version = "1.0.0"
+    @host = "api.example.com"
+    @description =:
+        | API description in Markdown.
+
+    /v1:
+
+        /users/{id<:int}:
+            GET (metadata <: bool [~header, ~optional, name="metadata"]):
+                | No description.
+                return
+
+    #---------------------------------------------------------------------------
+    # definitions
+"""
+
+OPENAPI_WITH_PATH_VAR_TYPE_OVERRIDDEN_IN_SECOND_METHOD = r"""
+openapi: "3.0"
+info:
+  title: Sample API
+  description: API description in Markdown.
+  version: 1.0.0
+host: api.example.com
+basePath: /v1
+schemes:
+  - https
+paths:
+  /users/{id}:
+    parameters:
+      - $ref: '#/parameters/id'
+      - in: header
+        name: metadata
+        schema:
+          type: boolean
+        required: false
+    get:
+      summary: Gets a user by ID
+      parameters:
+       -  in: header
+          name: metadata
+          schema:
+            type: string
+          enum:
+           - public
+           - personal
+           - all
+          required: true
+      responses:
+        '200':
+          description: OK
+    delete:
+      summary: Gets a user by ID
+      parameters:
+       -  in: path
+          name: id
+          schema:
+            type: string
+          required: true
+      responses:
+        '200':
+          description: OK
+parameters:
+  id:
+    in: path
+    name: id
+    schema:
+      type: integer
+    required: true
+    description: The user ID.
+"""
+
+OPENAPI_WITH_PATH_VAR_TYPE_OVERRIDDEN_IN_SECOND_METHOD_EXPECTED_SYSL = r"""
+ "Sample API" [package=""]:
+    @version = "1.0.0"
+    @host = "api.example.com"
+    @description =:
+        | API description in Markdown.
+
+    /v1:
+
+        /users/{id<:int}:
+            GET (metadata <: string [~header, ~required, name="metadata"]):
+                | No description.
+                return
+
+        /users/{id<:string}:
+            DELETE (metadata <: bool [~header, ~optional, name="metadata"]):
+                | No description.
+                return
+
+    #---------------------------------------------------------------------------
+    # definitions
+"""
+
+
+def getOutputString(input):
+    swag = yaml.load(input)
+    w = writer.Writer('sysl')
+    logger = FakeLogger()
+    t = OpenApiTranslator(logger)
+    t.translate(swag, appname='', package='', w=w)
+    return str(w), logger
+
+
+def test_importing_simple_openapi_with_json_tags():
+    output, _ = getOutputString(SIMPLE_OPENAPI_EXAMPLE)
```

Comment:
```python
def test_importing_simple_openapi_with_json_tags():
    output, _ = getOutputString(r"""\
        "openapi": "3.0"
        info:
          title: Simple
        paths:
          /test:
            get:
        ⋮
    """)
```

Reply:
Changed and deleted `SIMPLE_OPENAPI_EXAMPLE`. Why don't you want all the examples in one place?